client/src/containers/Dog/DogInfo.js

```diff
@@ -0,0 +1,45 @@
+// import {
+//     Card,
+//     CardContent,
+//     CardMedia,
+//     makeStyles,
+//     Typography,
+//   } from "@material-ui/core";
+import { useParams } from 'react-router-dom';
+import React, { useState, useEffect } from "react";
+import { DOGS_API } from "../../constants/api";
+import axios from "axios";
+
+  const DogInfo = () => {
+    let { dogId } = useParams();
+    const [dogData, setDogData] = useState({
+      name: "Loading",
+      image: "placeholder-dog.svg",
+
```

Comment:
Trailing empty line. Can set up **prettier** to automatically format and remove these blank lines on save: https://prettier.io/docs/en/rationale.html#empty-lines

Reply:
Prettier does this already on my thinkpad, suspect it wasn't set up properly on my tablet, maybe wasn't even enabled?